it/test/uk/gov/hmrc/alcoholdutycalculator/RatesIntegrationSpec.scala

```diff
@@ -16,16 +16,27 @@
 
 package uk.gov.hmrc.alcoholdutycalculator
 
+import com.fasterxml.jackson.databind.JsonNode
+import com.github.fge.jackson.JsonLoader
+import com.github.fge.jsonschema.core.report.ProcessingReport
+import com.github.fge.jsonschema.main.{JsonSchema, JsonSchemaFactory}
 import play.api.libs.json.Json
 import play.api.test.FakeRequest
 import uk.gov.hmrc.alcoholdutycalculator.base.ISpecBase
 import uk.gov.hmrc.alcoholdutycalculator.controllers.routes
 import uk.gov.hmrc.alcoholdutycalculator.models.{RateBand, RatePeriod, RateType}
 
+import java.io.InputStream
 import java.time.YearMonth
 
 class RatesIntegrationSpec extends ISpecBase {
 
+  private lazy val jsonSchemaFactory: JsonSchemaFactory = JsonSchemaFactory.byDefault
+  private lazy val jsonSchema: JsonSchema               =
+    jsonSchemaFactory.getJsonSchema(s"resource:/rates/alcohol-duty-rates-schema.json")
```

Comment:
Should we validate the file when we read it in RatesService? If yes, we can keep the schema in this folder but otherwise it will only be used for tests so we should maybe move it to the test resources.